wazo_acceptance/steps/asterisk.py

```diff
@@ -14,9 +14,14 @@
 def then_i_have_the_following_hints(context):
     output = context.helpers.asterisk.send_to_asterisk_cli('core show hints').split('\n')
     output = output[2:-3]  # strip header and footer
-    hints = [{'exten': line[:20].strip(), 'line': line[22:44].strip()}
-             for line in output]
+    hints = [{'exten': line[:20].strip(), 'line': line[22:44].strip()} for line in output]
 
     for row in context.table:
         row = row.as_dict()
         assert_that(hints, has_item(has_entries(row)))
+
+
+@then('Asterisk command "{command}" returns no error')
+def then_asterisk_command_group1_returns_no_error(context, command):
+    command = ['asterisk', '-rx', '"{command}"'.format(command=command)]
+    assert context.remote_sysutils.send_command(command)
```

Comment:
I don't get this assert, if I send `patate core reload` the assert pass

Reply:
I guess the point is to test if Asterisk is running correctly. If Asterisk is not running, the assert fails. But the assert is misleading, I'll make it clearer